oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/AuthorizedClientServiceReactiveOAuth2AuthorizedClientManager.java

```diff
@@ -121,6 +198,58 @@ public void setContextAttributesMapper(Function<OAuth2AuthorizeRequest, Mono<Map
 		this.contextAttributesMapper = contextAttributesMapper;
 	}
 
+	/**
+	 * Gets the handler that handles successful authorizations.
+	 *
+	 * @return the handler that handles successful authorizations.
+	 * @see DefaultAuthorizationSuccessHandler
+	 * @since 5.3
+	 */
+	public ReactiveOAuth2AuthorizationSuccessHandler getAuthorizationSuccessHandler() {
+		return authorizationSuccessHandler;
+	}
+
+	/**
+	 * Sets the handler that handles successful authorizations.
+	 *
+	 * <p>The {@link DefaultAuthorizationSuccessHandler default authorization success handler}
+	 * will save authorized clients in the {@link ReactiveOAuth2AuthorizedClientService}.</p>
+	 *
+	 * @param authorizationSuccessHandler the handler that handles successful authorizations.
+	 * @see DefaultAuthorizationSuccessHandler
+	 * @since 5.3
+	 */
+	public void setAuthorizationSuccessHandler(ReactiveOAuth2AuthorizationSuccessHandler authorizationSuccessHandler) {
+		Assert.notNull(authorizationSuccessHandler, "authorizationSuccessHandler cannot be null");
+		this.authorizationSuccessHandler = authorizationSuccessHandler;
+	}
+
+	/**
+	 * Gets the handler that handles authorization failures.
+	 *
+	 * @return the handler that handles authorization failures.
+	 * @see DefaultAuthorizationFailureHandler
+	 * @since 5.3
+	 */
+	public ReactiveOAuth2AuthorizationFailureHandler getAuthorizationFailureHandler() {
```

Comment:
We typically don't expose a getter for an optional collaborator - only the setter. However, we should javadoc the default collaborator.

Reply:
Removed